src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = (
+  {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  },
+) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
In case 'tabs' array is empty, trying to access 'tabs[0]' will return 'undefined', which will cause issues when accessing 'content'. Use optional chaining and nullish coalescing to handle this case.